src/microsim/schema/optical_config/filter.py

```diff
@@ -13,13 +16,28 @@ class _Filter(SimBaseModel):
     type: str
     name: str = ""
 
+    @computed_field  # type: ignore
+    @cached_property
+    def spectrum(self) -> Spectrum:
```

Comment:
note that this particular field is being added in #40 ... which will likely cause some conflicts here in this PR when you merge